lib/wickedbase.pm

```diff
@@ -489,17 +489,35 @@ sub setup_team {
     $self->get_from_data('wicked/ifcfg/ifcfg-eth0-hotplug',     $cfg_ifc0);
     $self->get_from_data('wicked/ifcfg/ifcfg-eth0-hotplug',     $cfg_ifc1);
     $self->get_from_data('wicked/teaming/ifcfg-team0-' . $mode, $cfg_team0);
-    file_content_replace($cfg_team0, ipaddr4 => $self->get_ip(type => 'host', netmask => 1), ipaddr6 => $self->get_ip(type => 'host6', netmask => 1), iface0 => $iface0, iface1 => $iface1);
+    my $ipaddr4  = $self->get_ip(type => 'host',  netmask       => 1);
+    my $ipaddr6  = $self->get_ip(type => 'host6', netmask       => 1);
+    my $ping_ip4 = $self->get_ip(type => 'host',  is_wicked_ref => 1);
+    my $ping_ip6 = $self->get_ip(type => 'host6', is_wicked_ref => 1);
+    file_content_replace($cfg_team0, ipaddr4 => $ipaddr4, ipaddr6 => $ipaddr6, iface0 => $iface0, iface1 => $iface1, ping_ip4 => $ping_ip4, ping_ip6 => $ping_ip6);
 
     $self->wicked_command('ifup', 'all');
 }
 
+sub get_team_active_link {
+    my ($self, $team_link) = @_;
+    my $out = script_output("teamdctl $team_link state view");
+    record_info('teamdctl', $out);
+    my @lines = split(/\n/, $out);
+    $out = join("\n", @lines[-1]);
```

Comment:
What is `@lines[-1]` ?

Reply:
Ah it's same as `$lines[-1]`, perl ...

Reply:
the last line of the multiline string, where the "active port" string is located.